src/pks/energy/constitutive_relations/thermal_conductivity/thermal_conductivity_surface_evaluator.cc

```diff
@@ -29,8 +29,8 @@ ThermalConductivitySurfaceEvaluator::ThermalConductivitySurfaceEvaluator(
 
   AMANZI_ASSERT(plist_.isSublist("thermal conductivity parameters"));
   Teuchos::ParameterList sublist = plist_.sublist("thermal conductivity parameters");
-  K_liq_ = sublist.get<double>("thermal conductivity of water [W/(m-K)]", 0.58);
-  K_ice_ = sublist.get<double>("thermal conductivity of ice [W/(m-K)]", 2.18);
+  K_liq_ = sublist.get<double>("thermal conductivity of water [W m^-1 K^-1]", 0.58);
+  K_ice_ = sublist.get<double>("thermal conductivity of ice [W m^-1 K^-1]", 2.18);
```

Comment:
Are we consistent? I think we are also using the old notation in the internal_energy_ice, "[J/mol-K]" for example.

Reply:
capillary_pressure_liq_ice is another place using "[J/kg]"

Reply:
Thank you.  I guess I have to go searching for more of these...

Reply:
ok, think I got them all...